File: app/src/main/java/com/example/user/SmartBartender/Ingredient.java
package com.example.user.SmartBartender;

class Ingredient {
    final int id;
    final int value;
    final int layer;
    final String name;

    public Ingredient(int value, String name, int id, int layer) {
        this.id = id;
        this.value = value;
        this.name = name;
        this.layer = layer;
    }
}

File: app/src/main/java/com/example/user/bartender/database/Ingredient.kt
package com.example.user.bartender.database

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "ingredient_table")
data class Ingredient(
        @PrimaryKey@ColumnInfo(name = "ingredient_name")
        val name: String,
        @ColumnInfo(name = "converter_coefficient")
        var c: Int
)
